index.js

```diff
@@ -239,6 +235,24 @@ async function addRelativeDeps({ paths, dev, script }) {
   })
 
   let pkg = getPackageJson()
+  let { config, filepath } = getConfig()
+
+  if (!filepath) {
+    console.error(
+      `[relative-deps][ERROR] You havn't initilized relative-deps yet. run "npx relative-deps init" and try again.`
+    )
+    process.exit(1);
+  }
+
+  // the other file formats will require complex parsing and it's not in the scope of work.
+  const allowedConfigFiles = ['package.json', 'relativeDependenciesrc', 'relativeDependenciesrc.json'];
```

Comment:
Looks like this block (248 - 255) of loading and validation config should be a function that is used wherever config is loaded, so for example at line 15 as well?